### ISSUE TITLE
Internal Pages Cannot be Accessed

### ISSUE THREAD

**Issue by ANT1H3R0** [pi-hole/pi-hole#1390]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*
- [X] The issue I'm reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**

_{0}_

---
**[FEATURE REQUEST | QUESTION | OTHER]:**

Please [submit your feature request here](https://discourse.pi-hole.net/c/feature-requests), so it is votable by the community.  It's also easier for us to track.

**[BUG | ISSUE] Expected Behaviour:**
I'm allowed to view the index page of my NGINX site with pihole installed.

**[BUG | ISSUE] Actual Behaviour:**
Access to the following site has been blocked: _

**[BUG | ISSUE] Steps to reproduce:**
Visit the site's index page.

- After following the Nginx configuration guide on this github, I was excited to finally have completed the setup of my webserver. However, upon attempting to visit the main page of my site, I found that it had been blocked by PiHole. I proceeded to check the blacklist and to whitelist my site's domain, all to no avail.

-I am aware of a similar issue posted previously. However, I do not have two IPs corresponding to my Pi (as far as I am aware.)

-This is true of all pages (excepting the admin page) on the site.

-Thank you in advance for your assistance.

**(Optional) Debug token generated by `pihole -d`:**

`melww28fjn`

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by technicalpyro** [comment]
try it with `URL/admin`

**Comment by ANT1H3R0** [comment]
I'd like to have an accessible site running on the server, as well.

**Comment by technicalpyro** [comment]
well if it is at the root it will be blocked as that is how pihole blocks things  unless you have changed something in the configs i would suggest checking the discourse forums as several users have successfully got this working as well.  i am not that familiar with ngmix so I am not sure but im sure others will have ideas

**Comment by DL6ER** [comment]
We change the error 404 page to the blocking page. Unfortunately, this can hide a number of errors. Please ensure that the page you want to reach is actually there.

Hint: When you access the device directly via its IP, Pi-hole *cannot* block anything (as it is running on the DNS level). If you still see the blocking page, you can be a 100% certain that you are facing a 404 (page not found) error, instead.

This behavior has been tweaked in the next version of Pi-hole to make such errors more obvious.

**Comment by ANT1H3R0** [comment]
I noticed that when looking around configs. However, I am entirely confident that the page I am attempting to view exists on the site. Regardless, I ended up running lighttpd on a different port, and using the proxy function of nginx to redirect /admin to the lighttpd server. This solution was found here: https://discourse.pi-hole.net/t/run-pi-hole-with-another-webservice/1903/2
By a user named deathbybandaid.
Thank you everyone for your assistance.
